versioning/scripts/dockerfiles/main.go

```diff
@@ -25,6 +27,40 @@ func writeDockerfile(version versions.Version, data []byte) {
 	check(err)
 }
 
+func verifyDockerfiles(spec versions.Spec, tmpl template.Template) {
+	foundDockerfile := make(map[string]bool)
+	failureCount := 0
+
+	for _, version := range spec.Versions {
+		data := renderDockerfile(version, tmpl)
+
+		path := filepath.Join(version.Dir, "Dockerfile")
+		dockerfile, err := ioutil.ReadFile(path)
+		check(err)
+
+		foundDockerfile[path] = true
+
+		if string(dockerfile) == string(data) {
+			log.Printf("%s: OK", path)
+		} else {
+			failureCount++
+			log.Printf("%s: FAIL", path)
+		}
+	}
+
+	err := filepath.Walk(".", func(path string, info os.FileInfo, err error) error {
```

Comment:
As it turns out, the PHP repo (on gerrit under limani-php) has `Dockerfile`s that are not included in `versions.yaml`. Let's remove this check for now. We can also extend `versions.yaml` to have an ignored list of `Dockerfile`s, but that doesn't have to be part of this PR.